packages/core/test/controls/radioGroupTests.tsx

```diff
@@ -81,4 +81,8 @@ describe("RadioGroup", () => {
         assert.lengthOf(group.find("address"), 1);
         assert.lengthOf(group.find(Radio), 2);
     });
+
+    function findInput(wrapper: ReactWrapper<any, any>, props: EnzymePropSelector) {
+        return wrapper.find("input").filter(props);
```

Comment:
this could also be done with `.hostNodes()`, didn't know about that when I fixed this suite.